requirements.txt

```
@@ -210,7 +210,7 @@ google-cloud-core==2.4.1
```

Verify the correct latest version of googleapis-common-protos. Based on the changelog, it should likely be 1.66.0 instead of 1.67.0.
*Suggested Fix*:
```suggestion
googleapis-common-protos==1.66.0
```